### ISSUE TITLE
Example SSR broken

### ISSUE THREAD

**Issue by Y-Less** [UnlyEd/next-right-now#9]
If you go to the examples with JS disabled, the main body shows, but not the header:

https://nrn-customer2.now.sh/

This is arguably fine as the header is only for developer information.  However, other parts also don't work such as the `Examples` link, and once on the examples page (by manually entering the address) you are greeted by:

![image](https://user-images.githubusercontent.com/991078/76114191-db52af00-5fe5-11ea-838c-ad1977d32d40.png)

Which is clearly not correctly server-side rendered.

**Comment by Vadorequest** [comment]
Good catch! This particular component is from Reactstrap `Alert`: https://reactstrap.github.io/components/alerts/

Maybe it doesn't support SSR. I hadn't noticed this, I'll take a look later. 
But it's not related to NRN in a whole, but specific to Reactstrap (maybe the lib isn't properly configured for SSR though).

As for links, I had noticed this behaviour, it's related to the Next.js framework. The links that are internal use some complex pre-fetching CSR and don't work without JS enabled, but I'm not sure if it's the expected behaviour, or an implementation issue. I'll dig in later as well.

To sum up: 
Everything is properly server side rendered, but some specific components aren't working as expected when JS is disabled.

**Comment by Vadorequest** [comment]
Issue opened at https://spectrum.chat/next-js/general/next-js-internal-links-dont-work-when-js-is-disabled-expected-or-a-bug~8130672a-21ac-4294-9c0b-2d0c6dd3e3d6

**Comment by Vadorequest** [comment]
Regarding the Reactstrap Alert component, I've checked with disabled JS and the bootstrap styles are properly imported. 
![image](https://user-images.githubusercontent.com/3807458/76117438-2243a300-5fec-11ea-811f-8b7adcc04afd.png)

I've looked into the DOM and everything is fine as well. Actually, the issue is because Bootstrap uses JS to display the Alert component at runtime, using a JS animation. It is hidden by default and then gets displayed. (opacity is changed from 0 to 1, that's all)

Of course, when JS is disabled then it's never displayed.

I could override this behaviour to display by default so that it displays correctly when JS is disabled, which will likely avoid thinking "SSR doesn't work". 😄 

![image](https://user-images.githubusercontent.com/3807458/76117677-87979400-5fec-11ea-9476-d5280c0fcc62.png)


![image](https://user-images.githubusercontent.com/3807458/76117691-8f573880-5fec-11ea-9409-682116e50257.png)

**Comment by Vadorequest** [comment]
Reopening because there is still the question about the links.

Reactstrap Alert has been fixed.

**Comment by Y-Less** [comment]
Thanks.  And yes, I wasn't entirely sure if this was the right place for the issue or not, but very impressed by the speed of your response anyway!

**Comment by Vadorequest** [comment]
@Y-Less  I've opened https://github.com/UnlyEd/next-right-now/issues/18 and I could use your feedback!
